content/en/docs/setup/install/multicluster/index.md

```diff
@@ -255,8 +226,9 @@ external attacks. Check with your cloud vendor to see what options are
 available.
 
 {{< text bash >}}
-$ CLUSTER=cluster1 NETWORK=network1 \
+$ MESH=mesh1 CLUSTER=cluster1 NETWORK=network1 \
     samples/multicluster/gen-eastwest-gateway.sh | \
```

Comment:
Here, and in other places that use use this script, I think it would be good to surround it in @ chars so that it's displayed as a clickable link. Less mysterious what it's doing.
```suggestion
    @samples/multicluster/gen-eastwest-gateway.sh@ | \
```